plugin/pulpcore/plugin/stages/declarative_version.py

```diff
@@ -12,7 +12,7 @@
 
 class DeclarativeVersion:
 
-    def __init__(self, first_stage, repository, sync_mode='mirror'):
+    def __init__(self, first_stage, repository, sync_mode='mirror', **kwargs):
```

Comment:
Please switch these to a named kwarg because that way the options show more explicitly in the options. So that would be s/**kwargs/inject_before=None, inject_after=None/. Since a variable in Python evaluates to False, you can then use:

```
self.inject_before = inject_before or []
self.inject_after = inject_after or []
```

Reply:
FYI, the time that I do use the kwargs idiom is whenever the arguments are passed through to super() and not used in this class directly.

Reply:
Thanks, updated.